### ISSUE TITLE
'shortnameToImage' is not exported by node_modules/emojione/lib/js/emojione.js

### ISSUE THREAD

**Issue by JiahaoShan** [jbw91/angular-emojione#1]
Hi, I have this issue when I tried to use the module.

> bundle failed: 'shortnameToImage' is not exported by node_modules/emojione/lib/js/emojione.js 
>             (imported by node_modules/angular-emojione/dist/angular-emojione.es5.js). For help fixing 
>             this error see https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-exported-by-module 

Any idea? Thanks

**Comment by johnbwoodruff** [comment]
The `shortnameToImage` function appears to be working correctly. I'm not sure how to reproduce your issue. Could you provide a repo that has this issue so I can further investigate?

**Comment by JiahaoShan** [comment]
Thanks for the reply. I am using the template from https://github.com/gangachris/githubionic and the package.json
```
{
  "name": "ionic-hello-world",
  "author": "Ionic Framework",
  "homepage": "http://ionicframework.com/",
  "private": true,
  "scripts": {
    "build": "ionic-app-scripts build",
    "watch": "ionic-app-scripts watch",
    "serve:before": "watch",
    "emulate:before": "build",
    "deploy:before": "build",
    "build:before": "build",
    "run:before": "build"
  },
  "dependencies": {
    "@angular/common": "2.2.1",
    "@angular/compiler": "2.2.1",
    "@angular/compiler-cli": "2.2.1",
    "@angular/core": "2.2.1",
    "@angular/forms": "2.2.1",
    "@angular/http": "2.2.1",
    "@angular/platform-browser": "2.2.1",
    "@angular/platform-browser-dynamic": "2.2.1",
    "@angular/platform-server": "2.2.1",
    "@ionic/storage": "2.0.1",
    "angular2-focus": "^1.1.0",
    "ionic-angular": "2.0.0-rc.4",
    "ionic-native": "2.2.3",
    "ionicons": "3.0.0",
    "rxjs": "^5.0.0-beta.12",
    "xml2json": "^0.11.0",
    "zone.js": "^0.8.4"
  },
  "devDependencies": {
    "@ionic/app-scripts": "0.0.36",
    "typescript": "2.0.10"
  },
  "cordovaPlugins": [
    "cordova-plugin-device",
    "cordova-plugin-console",
    "cordova-plugin-whitelist",
    "cordova-plugin-splashscreen",
    "cordova-plugin-statusbar",
    "ionic-plugin-keyboard"
  ],
  "cordovaPlatforms": [
    "ios",
    {
      "platform": "ios",
      "version": "",
      "locator": "ios"
    }
  ],
  "description": "githubIonic: An Ionic project",
  "config": {
    "ionic_source_map": "source-map"
  }
}

```

My Angular version is 2 rather than 4. I don't know if that may cause the error.

**Comment by johnbwoodruff** [comment]
@JiahaoShan I came up with the same issue when I tried it with your template. I then tried it with a project generated using the [Ionic CLI](http://ionicframework.com/docs/cli/) and it worked fine:

<img width="344" alt="screen shot 2017-04-21 at 9 09 07 pm" src="https://cloud.githubusercontent.com/assets/5883616/25301077/5053ebbc-26db-11e7-8af4-a1c82c90c447.png">

According to the link provided in the error message you posted, it looks like it has to do with the `rollup-plugin-commonjs` settings. Without debugging it for you it looks like a configuration issue with the starter you're using.

Also, related to your last comment, I am officially supporting Angular v4+. I haven't really tested it on Angular v2.
